python/ray/rllib/agents/impala/vtrace_policy_graph.py

```diff
@@ -35,7 +35,7 @@ def __init__(self,
                  bootstrap_value,
                  valid_mask,
                  vf_loss_coeff=0.5,
-                 entropy_coeff=-0.01,
+                 entropy_coeff=0.01,
```

Comment:
This one is missing a warning.

Reply:
Same for appo.

Reply:
I think the change in impala.py covers both.